make_mock_json.py

```diff
@@ -0,0 +1,315 @@
+
+        with open(pods_inpath, "r") as infile:
+            pods_data = json.load(infile)
+            image = "buildroot:latest"
+            pods_items = pods_data["items"] or []
+            for pod in pods_items:
+                pod_containers = pod["status"]["containerStatuses"] or []
+                for container in pod_containers:
+                    container["imageID"] = "docker-pullable://" + image
+                    container["image"] = image
+            self.comp_write("pods.json", pods_data)
+        os.remove(pods_inpath)
+
+    def create_mock_get_user(self):
+        self.osbs.get_user()
+        user_list = "get-users_~_-000.json"
+        user_list_path = "/".join([self.capture_dir, user_list])
+        with open(user_list_path, "r") as infile:
+            user_data = json.load(infile)
+            user_data["groups"] = []
+            user_data["identities"] = None
+            if user_data["fullName"]:
+                del user_data["fullName"]
+            user_data["metadata"]["name"] = "test"
+            user_data["metadata"]["selfLink"] = "/oapi/v1/users/test"
+            self.comp_write("get_user.json", user_data)
+        os.remove(user_list_path)
+
+    def create_a_mock_build(self, func, build_name, out_tag, build_args):
+        try:
+            build = func(**build_args)
+            build_id = build.get_build_name()
+            build = self.osbs.wait_for_build_to_get_scheduled(build_id)
+            self.osbs.watch_builds()
+            build = self.osbs.wait_for_build_to_finish(build_id)
+            self.osbs.get_build_logs(build_id)
+        except (subprocess.CalledProcessError, OsbsException):
+            pass
+
+        watch_data = canonize_data(copy.deepcopy(build.json), build_name, "Complete")
+        watch_obj = {
+            "object": watch_data,
+            "type": "MODIFIED"
+        }
+        out_name = "watch_build_test-" + out_tag + "build-123.json"
+        self.comp_write(out_name, watch_obj)
+
+        build_fname = "get-namespaces_osbs-stage_builds_" + build_id + "_-000.json"
+        build_path = "/".join([self.capture_dir, build_fname])
+        with open(build_path, "r") as infile:
+            build_data = canonize_data(json.load(infile), build_name, "Complete")
+            out_name = "build_test-" + out_tag + "build-123.json"
+            self.comp_write(out_name, build_data)
+        os.remove(build_path)
+
+    def create_mock_build(self):
+        build_kwargs = {
+            'git_uri': self.osbs.build_conf.get_git_uri(),
+            'git_ref': self.osbs.build_conf.get_git_ref(),
+            'git_branch': self.osbs.build_conf.get_git_branch(),
+            'user': self.osbs.build_conf.get_user(),
+            'release': TEST_BUILD,
+            'platform': "x86_64",
+            'arrangement_version': DEFAULT_ARRANGEMENT_VERSION,
+            'scratch': True,
+        }
+
+        self.create_a_mock_build(self.osbs.create_worker_build, TEST_BUILD, "", build_kwargs)
```

Comment:
maybe out_tag should be worker?

self.create_a_mock_build(self.osbs.create_worker_build, TEST_BUILD, "worker-", build_kwargs)